codes/extractive/netsum/netsum.py

```diff
@@ -185,6 +196,48 @@ def data_prep(text_path, summary_path, news_words_path, is_lemmatize, file_info,
             
     return X, Y
 
+def get_sent_word_sens(sentence):
+    '''
+    Get senses for each valid word in a sentence.
+    
+    Parameter:
+        sentence (str): a string containing a collection of words.
+        
+    Return:
+        wordSense (list): a list of extracted sense (default to be the first one)
+    '''
+    wordSense = []
+    for w in re.findall(r'[a-zA-Z]+', sentence):
+        if w in STOP_WORDS or len(w) < WORD_MIN_LEN:
+            continue
+        senses = wn.synsets(w)
+        if len(senses) == 0:
+            continue
+        wordSense.append(senses[0])
+    return wordSense
+
+def sentence_similarity(wordSense1, wordSense2):
+    '''
+    Calculating sentence similarity measurement.
+    
+    Parameters:
+        wordSense1 (list): a list of extracted sense for the first sentence.
+        wordSense2 (list): a list of extracted sense for the second sentence.
+        
+    Return:
+        the similarity score (float).
+    '''
+    similarity = 0.0
+    total = 0.0
+    for sense1 in wordSense1:
```

Comment:
check out the new version in lex rank, I use a dict to count senses to avoid duplicate work. Hope it helps.

Reply:
Will do in the next PR, thanks!